install/kubernetes/helm/istio/charts/grafana/templates/_affinity.tpl

```diff
@@ -0,0 +1,36 @@
+{{/* affinity - https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ */}}
```

Comment:
can you move this to `install/kubernetes/helm/istio/charts/_affinity.tpl`.  This will permit the deletion of all of the other `_affinity.tpl` files which are duplicated.  For an example of how this was done recently in master, please take a look at PR https://github.com/istio/istio/pull/3787.  See further review comments related to placing these config options in global.

Reply:
Yes, Okay for me.

Reply:
>In the templates/ directory, any file that begins with an underscore(_) is not expected to output a Kubernetes manifest file.

Have tried locally, we need to put `_affinity.tpl` in `install/kubernetes/helm/istio/templates/` directory.
It's OK to put this template definition file in parent chart directory, for templates in subcharts are compiled together with top-level templates.
Will include this change in next commit.